onadata/libs/tests/utils/test_csv_import.py

```diff
@@ -82,11 +87,13 @@ def test_submit_csv_and_rollback(self):
 
         count = Instance.objects.count()
         csv_import.submit_csv(self.user.username, self.xform, self.good_csv)
-        self.assertEqual(Instance.objects.count(),
-                         count + 9, u'submit_csv test Failed!')
+        self.assertEqual(Instance.objects.count(), count + 9,
+                         u'submit_csv test Failed!')
         # Check that correct # of submissions belong to our user
-        self.assertEqual(Instance.objects.filter(user=self.user).count(),
-                         count + 8, u'submit_csv username check failed!')
+        self.assertEqual(
+            Instance.objects.filter(user=self.user).count(),
+            count + 8,
```

Comment:
This has been around, was just applying https://github.com/google/yapf and `Isort` on files I touch.